### DISCUSSION TITLE
delete gen1 files

### DISCUSSION THREAD

**Discussion by AsertCreator** [CosmosOS/Cosmos#2520] [Q&A]
we have gen2 but gen1 files are still there, and they arent linked to any solution. maybe remove them or separate them into another repo?

**Comment by quajak** [comment]
I think we should be fine to delete them

**Reply by AsertCreator** [reply]
ok, ill remove them

**Reply by AsertCreator** [reply]
maybe we could also delete bochs binaries from resources folder. users should download bochs themselves ¯\_(ツ)_/¯. plus its outdated

**Reply by AsertCreator** [reply]
no, there are no gen1 files as i can find, maybe some trillion y.o. images that are in use

**Reply by quajak** [reply]
IIRC We need the bochs for easier autotesting with AppVeyor or Github actions.
